bundles.rst

```diff
@@ -45,9 +45,9 @@ Creating a Bundle
 -----------------
 
 This section creates and enables a new bundle to show there are only a few steps required.
-The new bundle is called AcmeTestBundle, where the ``Acme`` portion is just a
-dummy name that should be replaced by some "vendor" name that represents you or
-your organization (e.g. ABCTestBundle for some company named ``ABC``).
+The new bundle is called AcmeTestBundle, where the ``Acme`` portion is a dummy
```

Comment:
could we replace `dummy` with f.e. `example` or `placeholder` ?

Reply:
I merged to quickly. I've fixed this after the merge in 5717ed7c6]